meta-phosphor/common/recipes-kernel/linux/linux-obmc.bb

```diff
@@ -0,0 +1,26 @@
+DESCRIPTION = "Linux kernel for OpenBMC"
+SECTION = "kernel"
+LICENSE = "GPLv2"
+
+KBRANCH ?= "dev"
+
+inherit kernel
+require recipes-kernel/linux/linux-yocto.inc
+
+SRC_URI = "git://github.com/openbmc/linux;protocol=git;branch=${KBRANCH}"
+SRC_URI += "file://obmc-bsp.scc \
+            file://obmc-bsp.cfg \
+            file://obmc-bsp-user-config.cfg \
+            file://obmc-bsp-user-patches.scc \
+           "
+
+LINUX_VERSION ?= "4.3-rc1"
+LINUX_VERSION_EXTENSION ?= "-openbmc"
+
+SRCREV="${AUTOREV}"
+
+PV = "${LINUX_VERSION}+git${SRCPV}"
+
+COMPATIBLE_MACHINE_${MACHINE} = "openbmc"
```

Comment:
I could probably be convinced either way.  Why do you think it should be ast2400?

This variable has a partner - MACHINEOVERRIDES.  If you add openbmc to MACHINEOVERRIDES in an SOC config file, you are basically telling bitbake that this recipe is a candidate provider for the kernel package.  There isn't really anything here specific to ast2400, other than the kernel repo its pointing at.

I realize that right now this repo only "supports" ast2400, but in the longer term, we'll either have our own fork where we have all the support for the openbmc supported SOCs, or we'll just be pointing at kernel.org...but in either case, there isn't anything SOC specific.  So I was thinking ahead a little bit when I went this direction.